packages/sonataflow-deployment-webapp/src/pages/BasePage.tsx

```diff
@@ -0,0 +1,103 @@
+import {
+  Masthead,
+  MastheadBrand,
+  MastheadMain,
+  MastheadToggle,
+} from "@patternfly/react-core/dist/js/components/Masthead";
+import { Page, PageHeaderToolsItem, PageToggleButton } from "@patternfly/react-core/dist/js/components/Page";
+import { PageSidebar } from "@patternfly/react-core/dist/js/components/Page/PageSidebar";
+import { Text, TextContent, TextVariants } from "@patternfly/react-core/dist/js/components/Text";
+import { Tooltip } from "@patternfly/react-core/dist/js/components/Tooltip";
+import { Flex, FlexItem } from "@patternfly/react-core/dist/js/layouts/Flex";
+import { BarsIcon } from "@patternfly/react-icons/dist/js/icons";
+import HelpIcon from "@patternfly/react-icons/dist/js/icons/help-icon";
+import * as React from "react";
+import { useMemo } from "react";
+import { useHistory } from "react-router";
+import { useApp } from "../context/AppContext";
+import { routes } from "../routes";
+import { BasePageNav } from "./basePage/BasePageNav";
+
+export function BasePage(props: { children?: React.ReactNode }) {
+  const history = useHistory();
+  const app = useApp();
+
+  const masthead = useMemo(
+    () => (
+      <Masthead aria-label={"Page header"} display={{ default: "stack" }} className="app--masthead">
+        <MastheadMain style={{ justifyContent: "space-between" }}>
+          <PageHeaderToolsItem className={"pf-l-flex"}>
+            <MastheadToggle>
+              <PageToggleButton variant="plain" aria-label="Global NAV">
+                <BarsIcon />
+              </PageToggleButton>
+            </MastheadToggle>
+            <MastheadBrand
+              onClick={() => history.push({ pathname: routes.home.path({}) })}
+              style={{ textDecoration: "none" }}
+            >
+              <Brand className="kogito-tools-common--brand" src="favicon.svg" alt="Kie logo"></Brand>
+              <TextContent className="brand-name">
+                <Text component={TextVariants.h1}>
+                  {app.appDataPromise.status === PromiseStateStatus.PENDING ? "" : app.data.appName}
+                </Text>
+              </TextContent>
+            </MastheadBrand>
+          </PageHeaderToolsItem>
+          <Flex justifyContent={{ default: "justifyContentCenter" }}>
+            <FlexItem>
+              <PageHeaderToolsItem>
+                <Tooltip
+                  className="app--masterhead__disclaimer"
+                  position="bottom-end"
+                  key="disclaimer-tooltip"
+                  content={
+                    <>
+                      This deployment is intended to be used during <b>development</b>, so users should not use the
+                      deployed services in production or for any type of business-critical workloads.
+                    </>
+                  }
+                >
+                  <TextContent>
+                    <Text component={TextVariants.h5}>
+                      Development only
```

Comment:
I think showing/hiding the disclaimer can be another property for data.json, with `false` as default.
We already do this for Dashbuilder viewer (ref: `dashbuilder-viewer-deployment-webapp-data.json`)